.gitlab-ci.yml

```diff
@@ -0,0 +1,38 @@
+# Build merge request, push and tag
+workflow:
+  rules:
+    - if: $CI_MERGE_REQUEST_ID
+    - if: $CI_COMMIT_TAG
+    - if: $CI_COMMIT_BRANCH
+
+image: docker:git
+services:
+  - docker:dind
+
+before_script:
+  - apk add --update bash coreutils tar curl
+  - git clone --quiet --depth 1 https://github.com/ros-industrial/industrial_ci .industrial_ci -b master
+
+cache:
+  key: ${CI_JOB_NAME}
+  paths: 
+    - .ccache/
+
+variables:
+  TMPDIR: "${CI_PROJECT_DIR}.tmp"
+  DOCKER_RUN_OPTS: "-v /etc/ssl/certs:/etc/ssl/certs:ro"
+  CCACHE_DIR: "${CI_PROJECT_DIR}/.ccache"
+  CODE_COVERAGE: "codecov.io"
+  CODECOV_TOKEN: "36cef827-7894-4feb-b0d5-5c5c133cb341"
+
+foxy-moveit2:
+  variables:
+    ROS_DISTRO: "foxy"
+    DOCKER_IMAGE: "registry.gitlab.com/rosi-ap/rosiap_docker_images:moveit-foxy-nightly"
+    AFTER_INIT_EMBED: "source /opt/moveit2/install/setup.bash"
+  script: 
+    - echo "tLqbxEsTsUwtbjjwdBMd" | docker login -u rosi --password-stdin registry.gitlab.com
```

Comment:
Was this supposed to be merged in here? Or is that not a secret?

Reply:
Thanks, @gavanderhoorn. 

Noted on the requested change. @tanjpg Please make the appropriate amendment and update the pull request.

Reply:
Seeing as you've merged it, I don't believe you can update this PR any more.

That secret is in a few more places.

You should change the password on your `registry.gitlab.com` account as well I believe.

Reply:
@gavanderhoorn 

Apologies for this oversight. 

I have invalidated the password highlighted in this requested change. Will run another run of code review to address any further potential oversights before making a subsequent pull request. 

Thank you for pointing this out promptly. Will update once the new round of code review has completed.

Reply:
No problem, and no need to apologise.

Reply:
> That secret is in a few more places.
> 
> You should change the password on your `registry.gitlab.com` account as well I believe.

Hi @gavanderhoorn, thanks for pointing it out. This is actually a deployment token from gitlab that only have read access to the gitlab container registry, but not the source code or any write access. It is not a password to our repository. That said, I will move the ci docker to the registry under Github this Github repo.

Reply:
At least Github Actions has a nice way to pass secrets such as this one to CI scripts (even though it only allows read-only access, it's still a "secret"). Travis should have that as well (but has other problems at the moment.

You may want to look into those.